org.azymohliad.Qwertone.json

```diff
@@ -0,0 +1,44 @@
+{
+    "app-id": "org.azymohliad.Qwertone",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "master",
```

Comment:
the master runtime isn't available on flathub, this needs to be a released version

Reply:
thanks, done